docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic.TensorFlow
+{
+    class TextClassification
+    {
+        public const int MaxSentenceLenth = 600;
+        /// <summary>
+        /// Example use of the TensorFlow sentiment classification model.
+        /// Download the model from 
+        /// https://github.com/dotnet/machinelearning-testdata/blob/master/Microsoft.ML.TensorFlow.TestModels/sentiment_model
+        /// The model is in 'SavedModel' format. For further explanation on how was the `sentiment_model` created 
+        /// c.f. https://github.com/dotnet/machinelearning-testdata/blob/master/Microsoft.ML.TensorFlow.TestModels/sentiment_model/README.md
+        /// </summary>
+        public static void ScoringWithTextClassificationModelSample()
+        {
+            string modelLocation = @"sentiment_model";
```

Comment:
> string modelLocation = @"sentiment_model"; [](start = 11, length = 43)

``` public static void ScoringWithImageClassificationModelSample()
        {
            // Download the ResNet 101 model from the location below.
            // [https://storage.googleapis.com/download.tensorflow.org/models/tflite_11_05_08/resnet_v2_101.tgz](https://storage.googleapis.com/download.tensorflow.org/models/tflite_11_05_08/resnet_v2_101.tgz)
            var modelLocation = @"resnet_v2_101/resnet_v2_101_299_frozen.pb";
```
Can we be consistent with how we define model and information where to find it?

Reply:
I have made a method in SamplesUtils `SamplesUtils.DatasetUtils.DownloadTensorFlowSentimentModel()` to download the text model. However, the ResNet model is in .tgz format which needs decompressing. I don't find a good tool to do that in C#. So, leaving it as-is for now. Let me know if you have any recommendation?

---
In reply to: [254125969](https://github.com/dotnet/machinelearning/pull/2429#discussion_r254125969) [](ancestors = 254125969)